azure-core/azure/core/pipeline_client.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, base_url, config=None, pipeline=None):
             raise ValueError("Config is a required parameter")
         self.config = config
         self.base_url = base_url
-        self.pipeline = pipeline or self._build_pipeline(config, credentials)
```

Comment:
But there is no `self._build_pipeline()` method?
Maybe we should just make `pipeline` a required parameter?

Reply:
This is a good idea. I will make it required.